docs/conf.py

```diff
@@ -0,0 +1,352 @@
+# -*- coding: utf-8 -*-
+#
+# sphynx-demo documentation build configuration file, created by
+# sphinx-quickstart on Tue Aug  2 16:48:54 2016.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+import platform
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autosectionlabel','sphinxcontrib.plantuml','sphinx.ext.graphviz']
+
+if platform.system() == 'Windows':
+	plantuml = 'java -jar ..\utils\plantuml.jar'
+else:
+	plantuml = 'java -jar ../utils/plantuml.jar'
```

Comment:
Linux is not being able to load the extension. I propose to generate the images of the plantuml diagrams and add them instead of using the extension.

Reply:
After a Little research I discovered that a default sphinx installation includes several sphinxcontrib packages but that plantuml isn't one of them. Thus I've decided to remove the inline plantuml diagrams and replace them by images.